health-services/ExerciseSample/app/src/main/java/com/example/exercise/ExerciseService.kt

```diff
@@ -227,16 +218,13 @@ class ExerciseService : LifecycleService() {
                 else -> {
                 }
             }
-        } else if (oldState.isEnded && exerciseUpdate.state == ExerciseState.ACTIVE) {
+        } else if (oldState.isEnded && exerciseUpdate.exerciseStateInfo.state == ExerciseState.ACTIVE) {
             // Reset laps.
             _exerciseLaps.value = 0
         }
 
-        _exerciseState.value = exerciseUpdate.state
-        _exerciseMetrics.value = exerciseUpdate.latestMetrics
-        _aggregateMetrics.value = exerciseUpdate.latestAggregateMetrics
-        _exerciseDurationUpdate.value =
-            ActiveDurationUpdate(exerciseUpdate.activeDuration, Instant.now())
```

Comment:
I think this should be using `ActiveDurationCheckpoint` now?

Reply:
It should! Updated in c8a56b.